src/xcode/ENA/ENA/Source/Scenes/SelectValueViewController/SelectValueTableViewController.swift

```diff
@@ -48,6 +48,9 @@ final class SelectValueTableViewController: UITableViewController {
 		return cell
 	}
 
+	override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+		viewModel.cellViewModel(for: indexPath).rowHeight
+	}
```

Comment:
If the constraints are all set up correctly we should never need a manual cell height. Especially since we support Dynamic Type and multiple lines of text we will never know the height in pixels upfront.